english/models.py

```diff
@@ -31,27 +55,34 @@ def formfield(self, **kwargs):
         defaults.update(kwargs)
         return super(ArrayField, self).formfield(**defaults)
 
+    def to_python(self, value):
```

Comment:
Why you need it?

Reply:
To store integers in postresql arrayfield. 
Without that it raises an error